src/utils/cellSets/getCellClassProperties.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+
+/**
+ * Returns an array of properties of all the cellSets
+ * within the cellSetClass that cellId belongs to.
+ * @param {string} cellId The id of the cell we are searching for.
+ * @param {string} cellSetClassKey The cellSetClass we are searcing inside of
+ *  (e.g.: sample, louvain, scratchpad, etc..).
+ * @param {string} hierarchy CellSets hierarchy.
+ * @param {string} properties CellSets properties.
+ */
+
+const getCellClassProperties = (cellId, cellSetClassKey, hierarchy, properties) => {
```

Comment:
Return only the first matching cellSet properties sience we never use the other ones